### PR TITLE
Add read voucher commands to rollups-cli

### PR DESCRIPTION
**Pull request by GMKrieger**
#164

### CODE REVIEW

**Review by torives — DISMISSED** [review]

File: pkg/readerclient/voucher.go [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+import (
+	"context"
+	"fmt"
+
+	"github.com/Khan/genqlient/graphql"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+type Voucher struct {
+	// Notice index within the context of the input that produced it
```
Comment (gligneul) [inline pkg/readerclient/voucher.go:15]:
```suggestion
	// Voucher index within the context of the input that produced it
```

File: pkg/readerclient/voucher.go [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+	Index int `json:"index"`
+	// Input whose processing produced the notice
```
Comment (gligneul) [inline pkg/readerclient/voucher.go:17]:
```suggestion
	// Input whose processing produced the voucher
```

File: pkg/readerclient/voucher.go [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+	InputIndex int `json:"inputIndex"`
+	// Transaction destination address in Ethereum hex binary format (20 bytes), starting with '0x'
+	Destination string `json:"destination"`
+	// Notice data as a payload in Ethereum hex binary format, starting with '0x'
```
Comment (gligneul) [inline pkg/readerclient/voucher.go:21]:
```suggestion
	// Voucher data as a payload in Ethereum hex binary format, starting with '0x'
```

File: pkg/readerclient/voucher.go [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+	Payload hexutil.Bytes `json:"payload"`
+	// Proof object that allows this notice to be validated by the base layer blockchain
```
Comment (gligneul) [inline pkg/readerclient/voucher.go:23]:
```suggestion
	// Proof object that allows this voucher to be validated by the base layer blockchain
```

File: pkg/readerclient/voucher.go [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+type Voucher struct {
+	// Notice index within the context of the input that produced it
+	Index int `json:"index"`
+	// Input whose processing produced the notice
+	InputIndex int `json:"inputIndex"`
+	// Transaction destination address in Ethereum hex binary format (20 bytes), starting with '0x'
+	Destination string `json:"destination"`
```
Comment (gligneul) [inline pkg/readerclient/voucher.go:20]: This should be either hexutil.Bytes or common.Address

**Review by gligneul — CHANGES_REQUESTED** [review]

**Review by gligneul — APPROVED** [review]

**Review by torives — APPROVED** [review]

**Pull request merged**